pyproject.toml

```diff
@@ -14,3 +14,14 @@ include = '\.pyi?$'
 [tool.isort]
 profile = "black"
 line_length = 120
+
+[tool.pyright]
+include = ["keylime"]
+ignore = [
+    "keylime/backport_dataclasses.py",
+    "keylime/keylime_agent.py",
+    "keylime/elchecking/",
```

Comment:
The elchecking module should be verified in the long run, but it is fine to exclude it for this PR.